benches/my_benchmark.rs

```diff
@@ -214,11 +237,13 @@ pub fn bench_hrtf_panners() {
     osc.connect(&panner2);
     osc.start();
 
-    assert_eq!(ctx.start_rendering_sync().length(), SAMPLES);
+    assert_eq!(ctx.start_rendering_sync().length(), SAMPLES_SHORT);
 }
 
+#[cfg(feature = "iai")]
```

Comment:
maybe a macro would be justified to generate the code from here to the bottom of the file